addons/project_todo/__manifest__.py

```diff
@@ -30,6 +30,12 @@
             'project_todo/static/src/views/**/*',
             'project_todo/static/src/web/**/*',
         ],
+        'web.assets_tests': [
+            'project_todo/static/tests/tours/**/*',
+        ],
+        'web.qunit_suite_tests': [
+            'project_todo/static/tests/**/*',
```

Comment:
Oups sorry, I did not see it during my previous review. 🙈 

I think we should not include `tours/` in that assets.

```suggestion
            'project_todo/static/tests/helpers/**/*',
```